csclient/csclient.go

```diff
@@ -99,6 +100,11 @@ func (c *Client) DisableStats() {
 	c.statsDisabled = true
 }
 
+// SetHeader sets custom header to be sent to the charm store on each request.
```

Comment:
... sets custom HTTP headers that will be sent to the charm store
on each request.

?